configs/sst_insights.yaml

```diff
@@ -4,42 +4,6 @@ data:
   name: insights
   description: Insights
   maintainer: sst_insights
-  packages:
```

Comment:
This needs to be: `-  packages: []`

Reply:
oh ok, thanks for the notice, fixing

... maybe the `test_config_files.py` ought to check for this? ;-)